target/bin/delmailuser

```diff
@@ -1,69 +1,90 @@
 #! /bin/bash
+# shellcheck disable=SC2094
```

Comment:
would move the ignore line to the part that should actually be ignored

Reply:
This is done to disable this linting target for the whole file. Otherwise I would need to write it twice (and at one place where it somehow doesn't really work, Bash is complaining somehow, idk why). What do you think?

Reply:
Regardless of where it is I would like to have a short comment that says in plain text why it is needed.

Reply:
Good idea. I have added it with the latest commit.